web/src/lib/components/admin-page/settings/image/image-settings.svelte

```diff
@@ -30,6 +30,7 @@
           bind:value={config.image.thumbnailFormat}
           options={[
             { value: ImageFormat.Jpeg, text: 'JPEG' },
+            { value: ImageFormat.Jxl, text: 'JPEG XL (not supported by Google Chrome)' },
```

Comment:
It also doesn't work in Firefox, Edge, or any other browser besides Safari 
```suggestion
            { value: ImageFormat.Jxl, text: 'JPEG XL (only supported by Safari)' },
```

Reply:
It is supported by Firefox as an option. Please see https://www.phoronix.com/news/Mozilla-Neutral-On-JPEG-XL (https://github.com/martinthomson/standards-positions/commit/bf142e30846d38af18651d539ddcecba2716cae8).

Reply:
Hmmm, it doesn't work in Firefox even with `image.jxl.enabled=true`. I will change to the suggested wording.